app/index.html

```diff
@@ -308,6 +303,15 @@ <h1 ng-bind="globalPage.title"></h1>
   </script>
   <!-- /build -->
 
-  <script type="text/javascript">_satellite.pageBottom();</script>
+  <!-- Adobe DTM -->
+  <script>
+    window.digitalData = {};
+  </script>
+  <!-- build:remove:dist -->
```

Comment:
This is currently an issue from Josh's initial implementation but both of these scripts are available on dev when they shouldn't be. I'm not sure of a good way to fix it with [grunt-processhtml](https://www.npmjs.com/package/grunt-processhtml). Maybe we could comment them out and then remove the comments but that seems messier...